migration/mapper.py

```diff
@@ -0,0 +1,636 @@
+
+
+z={}
```

Comment:
in the IDE you use isnt there a prettier like format on save etc? having random spaces patter is very jarring to the eye.